packages/components/pages/f-account-info/package.json

```diff
@@ -40,20 +40,22 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-globalisation": "1.0.0"
+    "@justeat/f-card-with-content": "1.0.0",
```

Comment:
Should this one be a dev dependencies

Reply:
No it's a [molecule](https://github.com/justeat/fozzie-components/tree/master/packages/components/molecules/f-card-with-content) component that is required. I think `f-card` and `f-button` are missing though?

Reply:
@AndyMayJE Do you not need to add these packages?

Reply:
Thanks thought I had added these